src/dos/program_imgmount.cpp

```diff
@@ -235,10 +235,15 @@ void IMGMOUNT::Run(void)
 		return;
 	}
 
+	auto used_wildcards = false;
```

Comment:
`use_wildcards`? (present)

Reply:
How about `has_wildcards`?  (it's past tense; we're checking if the user used wildcards in their CLI statement.)

I think `has_` is a good fit w/ other bools.

Reply:
Yes "has" is the best, agreed.